src/root-config.ts

```diff
@@ -1,13 +1,13 @@
 import { registerApplication, start } from "single-spa";
 import * as activityFunctions from "./routes";
 import "./register-shared-deps";
+import { registerAllCoreApplications } from "@openmrs/esm-root-config";
 
 export const coreApps = {
```

Comment:
nit: consider using another name other than coreApps to distinguish the "coreApps" in esm-root-config.  Perhaps ampathApps

Reply:
👍 agreed

Reply:
Also, I would recommend using the same pattern as done for esm-root-config where ampathApps exist in a separate file along with the associated activity function. See https://github.com/openmrs/openmrs-esm-root-config/blob/master/src/single-spa-applications/core-applications.js